src/Tribe/Attendees.php

```diff
@@ -555,15 +556,15 @@ public function render() {
 	 *
 	 * @since 4.6.2
 	 *
-	 * @param $event_id
+	 * @param mixed $event_id The ID of the event to export the list for or 'all' for all events.
```

Comment:
Minor: `mixed` is accurate, but since we're only expecting 2 types we could do it this way:

```suggestion
	 * @param int|string $event_id The ID of the event to export the list for or 'all' for all events.
```

Same for the filter below.

Reply:
I should also say the same applies for the other places where we're expecting `all` or an event ID in the file.